.github/workflows/vvv-provisioning.yml

```diff
@@ -0,0 +1,91 @@
+on:
+  push:
+    branches: [ develop ]
+  pull_request:
+    branches: [ develop, stable ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    name: Run Provisioner
+    # The type of runner that the job will run on
+    runs-on: ubuntu-18.04
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      # Make Symlinks
+      - name: Create Vagrant Like Environment
+        run: |
+          # uninstall pre installed packages (to test if utilities work)
+          sudo apt-get -q --autoremove --purge remove php*
+          sudo apt-get -q autoclean
+
+          # create vagrant user
+          sudo groupadd -g 2000 vagrant
+          sudo useradd -u 2000 -g vagrant -m vagrant
+
+          # Install bindfs, which would allow us to "mount" folders with specific permissions
+          # similar to vagrant shares
+          sudo apt-get update
+          sudo apt-get install bindfs -y
+
+          # vbox_mount function to sumulate synced folders
+          function vbox_mount() {
+            sudo mkdir -p "$2"
+            sudo bindfs --chown-ignore --chgrp-ignore --chmod-ignore -u "$3" -g "$4" "$1" "$2"
+          }
+
+          # create srv folder
+          sudo -u "vagrant" mkdir -p "/srv"
+
+          # copy files provided by vagrant
+          sudo cp -f "$GITHUB_WORKSPACE/config/default-config.yml" "$GITHUB_WORKSPACE/config/config.yml"
+          sudo cp -f "$GITHUB_WORKSPACE/version" "/home/vagrant/version"
+
+          # simulate vagrant synced folders
+          sudo mkdir -p "$GITHUB_WORKSPACE/certificates"
+          sudo mkdir -p "$GITHUB_WORKSPACE/log/memcached"
+          sudo mkdir -p "$GITHUB_WORKSPACE/log/nginx"
+          sudo mkdir -p "$GITHUB_WORKSPACE/log/php"
+          sudo mkdir -p "$GITHUB_WORKSPACE/log/provisioners"
+          vbox_mount "$GITHUB_WORKSPACE/database/sql" "/srv/database" "vagrant" "vagrant"
+          vbox_mount "$GITHUB_WORKSPACE/config" "/srv/config" "vagrant" "vagrant"
+          vbox_mount "$GITHUB_WORKSPACE/provision" "/srv/provision" "vagrant" "vagrant"
+          vbox_mount "$GITHUB_WORKSPACE/certificates" "/srv/certificates" "vagrant" "vagrant"
+          vbox_mount "$GITHUB_WORKSPACE/www" "/srv/www" "vagrant" "www-data"
+          vbox_mount "$GITHUB_WORKSPACE/log/memcached" "/var/log/memcached" "root" "syslog"
+          vbox_mount "$GITHUB_WORKSPACE/log/nginx" "/var/log/nginx" "root" "syslog"
+          vbox_mount "$GITHUB_WORKSPACE/log/php" "/var/log/php" "root" "syslog"
+          vbox_mount "$GITHUB_WORKSPACE/log/provisioners" "/var/log/provisioners" "root" "syslog"
+
+      # Runs the main provisioner as sudo
+      - name: Run all provisioners
+        run: sudo bash /srv/provision/tests/run-provisioners.sh
```

Comment:
If it's possible I'd like some of the provisiooners to be their own steps so we can see froom GH Actions which one failed, otherwise it'll always fail on `Run all provisioners` requiring us to look through all the artefacts

Reply:
I'll look into separating this